### DISCUSSION TITLE
Problems integreating components : [W113] Sourced component 'ner' may not work as expected: source vectors are not identical to current pipeline vectors.

### DISCUSSION THREAD

**Discussion by Alvaro8gb** [explosion/spaCy#11003] [Help: Coding & Implementations] [answered]
I have trained a nlp pipeline with ner component and I want to integrate it with es_core_news_lg. But they said that the vectors are diferents. How can I only change the component of ner in one pretrained spacy's model. 

Working in spacy v3.3.
Code that shows the warning : 
``` 
import spacy

path_my_model = "./my_model"

nlp = spacy.load(path_my_model)

nlp_es = spacy.load("es_core_news_lg")

nlp_es.remove_pipe("ner")

nlp_es.add_pipe("ner",source=nlp, last=True)

doc = nlp_es("Mi nombre es Alvaro")

for token in doc:
    print(token.text, token.pos_, token.dep_)

for ent in doc.ents :
    print(ent.text, ent.start_char, ent.end_char, ent.label_)
```

**Comment by polm** [comment, marked as answer]
If you change `vectors` in `[paths]` to `es_core_news_lg` and train your model again, your model will be trained with the same word vectors as the other model, and should work. 

Note your Markdown formatting didn't work, please see the [Github Markdown guide](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-code).

**Reply by Alvaro8gb** [reply]
Thanks so much Polm. I will try it now.
